examples/imitation_learning_openbot/generate_dataset.py

```diff
@@ -166,22 +167,22 @@
     
                 # send control action to the agent and collect observations
                 obs, _, _, env_step_info = env.step(action={"set_duty_cycles": action})
-    
+
+                observation_components_to_modify = { "final_color": ["camera.final_color"] }
```

Comment:
Place this code inside the `if args.debug` and again in `if not args.benchmark`. It costs an extra line of code but it makes the if statements less coupled.